examples/lab_testing_CU/lab_test_P_sample.i

```diff
@@ -4,7 +4,7 @@
 pconf = 8 #MPa --> the initial effective confining pressure applied to the sample before shearing
 pw = 9 # the initial pore pressure after consilidation phase
 pconf_total = ${pconf}+${pw} #MPa Total confining presure
-dip_angle = 90
```

Comment:
dip_angle=90 for P sample is correct. But, for S-Sample it should be 0, which is wrong in S sample example. Could you please correct it?

Reply:
Sure, done.